subsys/bluetooth/host/audio/mcc.c

```diff
@@ -1758,6 +1824,37 @@ int bt_mcc_read_current_track_obj_id(struct bt_conn *conn)
 	return err;
 }
 
+int bt_mcc_set_current_track_obj_id(struct bt_conn *conn, uint64_t obj_id)
+{
+	int err;
+
+	if (!conn) {
```

Comment:
Prefer to use CHECKIF for static checks such as NULL checks so that it can easily be optimized later by the application if wanted. 
```suggestion
	CHECKIF(!conn) {
```

Reply:
Added to later task